Documentation/ceph-cluster-crd.md

```diff
@@ -1385,6 +1385,9 @@ By default, the Key Encryption Keys (also known as Data Encryption Keys) are sto
 However, if a Key Management System exists Rook is capable of using it. HashiCorp Vault is the only KMS currently supported by Rook.
 Please refer to the next section.
 
+RGW supports encryption via KMS using HashiCorp Vault. If below settings are defined, then RGW establish connection between Vault and
```

Comment:
It would be nice to link the official RGW doc for more details?

Reply:
I have the link at the end of the doc, it makes sense to add here